dtm0/fop.c

```diff
@@ -369,28 +392,36 @@ static int dtm0_fom_tick(struct m0_fom *fom)
 
 		if (m0_dtm0_in_ut() && req->dtr_msg == DMT_EXECUTE &&
 		    m0_dtm0_is_a_persistent_dtm(fom->fo_service)) {
-			rc = m0_dtm0_tx_desc_copy(&req->dtr_txr,
-						  &rep->dr_txr);
-			M0_ASSERT(rc == 0);
+			rc = m0_dtm0_tx_desc_copy(&req->dtr_txr, &rep->dr_txr);
+			if (rc != 0)
+				m0_fom_phase_move(fom, M0_ERR(rc), M0_FOPH_FAILURE);
 			rc = m0_dtm0_send_msg(fom, DMT_EXECUTED,
 					      &req->dtr_txr.dtd_id.dti_fid,
 					      &req->dtr_txr);
-			M0_ASSERT(rc == 0);
+			if (rc != 0)
```

Comment:
Some of these line are longer than 80 characters. Let's try to see if we can fix it. If looks like we too many levels of nested if/else-s here, I think we could try to reduce the amount of levels to improve readability of the code and to make this code aligned with the coding style.

Reply:
OK. Will do.